docs/source/core_services/service_agents/emailer/EmailerAgent.rst

```diff
@@ -0,0 +1,29 @@
+.. _EmailerAgent:
+
+===============
+Emailer Agent
+===============
+Emailer agent is responsible for sending emails for an instance. It has been written so that any agent on the instance
+can send emails through it via the "send_email" method or through the pubsub message bus using the topic
+"platform/send_email".
+
+By default any alerts will be sent through this agent. In addition all emails will be published to the
+"record/sent_email" topic for a historian to be able to capture that data.
+
+Configuration
+~~~~~~~~~~~~~
+A typical configuration for this agent is as follows. We need to specify the SMTP server address or the mail gateway,
+email address of the sender, email addresses of all the recipients and frequency (in minutes) with which the emails need
+to be sent.
+
```

Comment:
The allow frequency is for not allowing many alerts to be sent so the way you put it it's only send it for the frequency...when in fact it will only allow a specific keyed email to be sent once over that timespan...

in the example if you had an alert come in 2 times within the 10 minutes only one would be sent.